terraform/addons/monitoring/lambda/main.go

```diff
@@ -37,27 +37,40 @@ import (
 	flags "github.com/jessevdk/go-flags"
 )
 
-type NullEvent struct{}
+type (
+	NullEvent       struct{}
+	SNSTopicArnsMap map[string]string
```

Comment:
Created a map to hold the different topics used for cron system vs cron schedule failure alerts.